plugins/goodreads.koplugin/goodreadsbook.lua

```diff
@@ -190,17 +190,19 @@ function GoodreadsBook:genBookInfoGroup()
         align = "top",
         HorizontalSpan:new{ width =  split_span_width }
     }
-    --thumbnail
+    -- thumbnail
     local http = require("socket.http")
     local body = http.request(self.dates.image)
     local image = false
     if body then image = Pic.openJPGDocumentFromMem(body) end
     if image then
         table.insert(book_info_group, ImageWidget:new{
-            image = image.image_bb,
+            image_disposable = false,
+            image = image.image_bb:copy(),
```

Comment:
This doesnt look right to me: if you're making a copy, you would want ImageWidget to do the cleanup, which it won't do if image_disposable=false (default is image_disposable=true : it can dispose of it, and so will do the cleanup).